images/build/installDotNetCore.sh

```diff
@@ -18,6 +18,9 @@ if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set +x
     sasToken=$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN
     set -x
+    if [ -z "$sasToken" ]; then
+      echo "sasToken cannot be empty for $sdkStorageAccountUrl."
+    fi
```

Comment:
yah, we do.
I just added this condition which prints if the `sasToken` is empty. Now, it will be easier to realize/debug if our pipelines fail because of the `sasToken` related issue. 
Previously, there was no clue given in the logs. So, it was hard to find out the root-cause of the problem. As, I had seen this a number of times, it became familiar to me. But if anyone new faces the same problem, it would feel a nightmare to find and fix it.

This condition should ease the pain. :D